ci-operator/step-registry/assisted/baremetal/operator/ztp/assisted-baremetal-operator-ztp-workflow.yaml

```diff
@@ -19,6 +19,7 @@ workflow:
       USER_DATA_FILENAME: "user-data"
       DISCONNECTED: "false"
       NUM_EXTRA_WORKERS: "1"
+      GOVERSION: "1.21"
```

Comment:
```suggestion
      GOVERSION: "1.21.7"
```
looks like there's no "1.21" version in the mirror :thinking: 
https://storage.googleapis.com/golang/go1.21.linux-amd64.tar.gz

Reply:
Thanks for catching that! Modified it to the version suggested